index.html

```diff
@@ -102,7 +102,7 @@ <h3>5 Must Know Interview Questions for Javascript!</h3>
                         <p>307,739 views <span>3 years ago</span></p>
                     </div>
                 </div>
-                <div class="video">
+                <section class="video">
                     <div class="content--video">
                         <iframe width="172" height="95" src="https://www.youtube.com/embed/w47sY1sCWco" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
                     </div>
```

Comment:
Kindly consider using 2 tab spaces as indentation. It makes your code more readable. 
Also consider formatting your code to make it easy to read and catch bugs where there are bugs